server/src/main/java/tabletop/domain/exceptions/ErrorInfo.java

```diff
@@ -0,0 +1,21 @@
+package tabletop.domain.exceptions;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+public enum ErrorInfo {
```

Comment:
check out ValidationMessages, very similar thing but also works with automatic validation with the Valid annotation. I would delete this ErrorInfo class and use validation messages instead.